bin/tachyon

```diff
@@ -18,6 +18,9 @@ function printUsage {
   echo -e "  runTests      \t Run all end-to-end tests on a Tachyon cluster."
   echo -e "  journalCrashTest\t Test the Master Journal System in a crash scenario. Try 'tachyon journalCrashTest -help' for more help."
   echo -e "                  \t NOTE: This command will stop the existing server and creates a new one!"
+  echo -e "  masterConcurrencyPerf\t Performance tests for concurrent master operations."
+  echo -e "                       \t NOTE: This command will restart a new Tachyon cluster locally with one master and one worker,"
+  echo -e "                       \t and underfs storage directory will be deleted and recreated, journal directory will be reformatted."
```

Comment:
Why can't the benchmark run on an existing Tachyon cluster and write the files to a directory like `/masterConcurrencyPerf`?